daras_ai_v2/language_model.py

```diff
@@ -143,6 +147,7 @@ class LargeLanguageModels(Enum):
         llm_api=LLMApis.openai,
         context_window=4096,
         price=1,
+        supports_json=True,
```

Comment:
Have you tested this? When I went through and tested all the open ai models, this one did not work for me 🤔

Reply:
I just copied the ones listed here - https://platform.openai.com/docs/guides/json-mode

Reply:
in any case it should work for all models to some degree because of the added json prompt

Reply:
Yes, but when I run gpt 3.5 with the json flag enabled, it errors out in practice: https://gooey.ai/compare-large-language-models/?run_id=oygoftsplu02&uid=2pZJQouaP2Okm18cbEhIpOQpqCg1

Reply:
i guess not though, it might lead to a misuse of json mode on non json mode models

Reply:
The supports_json flag above was meant to be used to disable json mode for models that don't support it. I'll leave it up to you whether we prefer showing the JSON format ui but throwing an error in that case (or perhaps even implement our own JSON mode that retries a couple of times if the response happens to be invalid json) over ignoring it for unsupported models

Reply:
What we have now is ok. 

PS doing json mode well requires constrained decoding https://lmsys.org/blog/2024-02-05-compressed-fsm/